LamestWebserver/LamestWebserver/HttpRequest.cs

```diff
@@ -449,8 +457,13 @@ private static HttpRequest GetCookiesAndModified(HttpRequest packet, string[] li
                 {
                     packet.IsWebsocketUpgradeRequest = true;
                 }
+                else if(packet.Range == null && linput[i].Contains("Range: bytes="))
```

Comment:
There should be a `string.StartsWith(string)` method, that would fit this use case